ci-operator/step-registry/gcp/provision/min-permissions/gcp-provision-min-permissions-commands.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+GOOGLE_PROJECT_ID="$(< ${CLUSTER_PROFILE_DIR}/openshift_gcp_project)"
+export GCP_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/gce.json"
+sa_email=$(jq -r .client_email ${GCP_SHARED_CREDENTIALS_FILE})
+if ! gcloud auth list | grep -E "\*\s+${sa_email}"
+then
+  gcloud auth activate-service-account --key-file="${GCP_SHARED_CREDENTIALS_FILE}"
+  gcloud config set project "${GOOGLE_PROJECT_ID}"
+fi
+
+if [[ "${INSTALLATION_METHOD}" == "IPI" ]]; then
+  iam_account="ipi-min-permissions-sa@${GOOGLE_PROJECT_ID}.iam.gserviceaccount.com"
```

Comment:
Because the iam_account is pre-configured, and existing in the specific cluster profile account, can you give some comments for it to tell others some info about it, e.g: how to create it, where is the document for it.